docs/developer/extending/apps.mdx

```diff
@@ -1,8 +1,461 @@
+
+
+### Webhooks
+
+Webhooks can be used to receive data from Saleor when particular events happen. The available events are:
+
+| Name                        | Description                                                                                                                      |
+| --------------------------- | -------------------------------------------------------------------------------------------------------------------------------- |
+| `ANY_EVENTS`                | Receive webhooks when any of the available events are triggered (wildcard event).                                                |
+| `CHECKOUT_QUANTITY_CHANGED` | Quantity of a checkout line is changed.                                                                                          |
+| `CUSTOMER_CREATED`          | A new customer account is created.                                                                                               |
+| `ORDER_CREATED`             | A new order is placed.                                                                                                           |
+| `ORDER_FULLY_PAID`          | Payment is made and an order is fully paid.                                                                                      |
+| `ORDER_UPDATED`             | An order is updated; triggered for all changes related to an order; covers all other order webhooks, except for `ORDER_CREATED`. |
+| `ORDER_CANCELLED`           | An order is cancelled.                                                                                                           |
+| `ORDER_FULFILLED`           | An order is fulfilled.                                                                                                           |
+| `FULFILLMENT_CREATED`       | A new fulfillment is created.                                                                                                    |
+| `PRODUCT_CREATED`           | A new product is created.                                                                                                        |
+
+Webhooks' payloads are sent in `POST` requests.
+
+### Difference between `create` and `install`
+- `create`: create a custom App instance, assign explicitly provided permissions. 
+The app is fully manageable. You can change assigned permissions, add webhooks, or auth_token. 
+`create` can be useful for creating integration with some services which will parse the webhook payload.
+- `install`: create App instance and prepare communication channel. 
+This approach uses a defined App manifest to prepare all required objects. 
+This approach is recommended for creating integration with a service dedicated to Saleor. 
+Installed App can be activated, deactivated, deleted but cannot be modified by using a mutation dedicated to `create` actions. 
```

Comment:
"cannot be modified by using a mutation dedicated to `create` actions." - Why? Can we name these?